mne/export/_eeglab.py

```diff
@@ -17,8 +17,11 @@ def _export_raw(fname, raw):
 
     # remove extra epoc and STI channels
     drop_chs = ['epoc']
-    if not (raw.filenames[0].endswith('.fif')):
-        drop_chs.append('STI 014')
+    try:
+        if not (raw.filenames[0].endswith('.fif')):
+            drop_chs.append('STI 014')
+    except AttributeError:  # mne.io.RawArray has no filenames attribute
+        pass
```

Comment:
I think I'd be more comfortable with:

```suggestion
    # RawArray doesn't have `filenames` attribute
    if hasattr(raw, 'filenames') and not raw.filenames[0].endswith('.fif'):
        drop_chs.append('STI 014')
```

the `except Error: pass` pattern always makes me nervous that silent bugs might slip through

Reply:
In fact, there is a misunderstanding by my comment.
RawArray has filenames attribute but RawArray.filenames[0]==None.